topics/terraform/index.md

```diff
@@ -0,0 +1,19 @@
+---
+created_by: Mitchell Hashimoto, HashiCorp
+display_name: Terraform
+github_url: https://github.com/hashicorp/terraform
+logo: terraform.png
+related: hashicorp, hashicorp-terraform, infrastructure, infrastructure-as-code, terraform-configurations, terraform-module, terraform-modules, terraform-providers
+released: July 28, 2014
+short_description: An infrastructure as code tool for building, changing, and versioning infrastructure safely and efficiently.
```

Comment:
I think "infrastructure as code" should be hyphenated as a compound adjective describing "tool."